sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md

```diff
@@ -1,16 +1,9 @@
 # Release History
 
-## 1.1.0 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.1 (Unreleased)
```

Comment:
release date?

Reply:
oh unless you're not planning on not releasing today

Reply:
was going to leave it and release sometime next week :)